scripts/lsp_types.lua

```diff
@@ -18,24 +19,117 @@ local function tofile(fname, text)
   end
 end
 
-function M.gen(opt)
-  if vim.uv.fs_stat('./lsp.json') then
-    vim.fn.delete('./lsp.json')
+local function sort_by_method(tbl)
```

Comment:
This is unnecessarily complicated. Just sort by the _generated_ name.

I pushed an update that does that.